addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -716,7 +719,7 @@ function factory(dependencies) {
             // If the current user invited a new user, and the new user is
             // connecting for the first time while the current user is present
             // then open a chat for the current user with the new user.
-            this.env.services['bus_service'].sendNotification(title, message);
+            this.env.services['bus_service'].sendNotification({ title: title, message: message });
```

Comment:
Alphabetical order for keys in object, and no need to repeat the key when it's the same name as variable
```suggestion
            this.env.services['bus_service'].sendNotification({ message, title });
```